### PR TITLE
Lh/ui updates

### LINKED ISSUE #633: Feedback - UI
CB
Carlos Barahona
Additional comments•2019-06-03 11:15:24 AM
received from: crbarahona@ucdavis.edu

Original Subject : Add Equipment Control
Urgency Level : Non-Critical Issue
Support Department : Programming Support
For Application : Peaks

Supplied Message Body :
The "Add Equipment" control on the Equipment page is non-intuitive. First of all, the control shouldn't only be located on the bottom of the page. While this is apparent with a short or empty list, with a populated equipment inventory, it adds a (IMO) non-trivial step the the process.
Additionally, the control ACTUALLY opens the "Assign Equipment" modal, not the "Add Equipment" one. I'm aware you can THEN create new equipment, but the control should be more appropriately labeled.

### PR DESCRIPTION
**Pull request by laholstege**
closes #633 
moves button for adding for all asset types
changes wording to be "add equipment" instead of "assign equipment" when clicking add button
image

### CODE REVIEW

File: Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx [hunk 1]
```diff
@@ -77,15 +77,17 @@ export default class EquipmentContainer extends React.Component<IProps, IState>
 
         const commonAttributeKeys = await this.context.fetch(attrFetchUrl);
 
-        const equipmentTypeFetchUrl = `/api/${this.context.team.slug}/equipment/ListEquipmentTypes/`;
+        const equipmentTypeFetchUrl = `/api/${
+            this.context.team.slug
+        }/equipment/ListEquipmentTypes/`;
```
Comment (jSylvestre) [inline Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx]: Don't like the white space, but doesn't make a difference

Comment (laholstege) [reply Keas.Mvc/ClientApp/components/Equipment/EquipmentContainer.tsx]: blame prettier, not me 🤷‍♀

File: Keas.Mvc/ClientApp/components/Access/AccessContainer.tsx [hunk 2]
```diff
@@ -74,24 +74,24 @@ export default class AccessContainer extends React.Component<IProps, IState> {
         return (
             <div className="card access-color">
                 <div className="card-header-access">
-                    <div className="card-head">
+                    <div className="card-head row justify-content-between">
```
Comment (srkirkland) [inline Keas.Mvc/ClientApp/components/Access/AccessContainer.tsx]: flex it!

Comment (laholstege) [reply Keas.Mvc/ClientApp/components/Access/AccessContainer.tsx]: 💪

**Pull request merged**